cmd/k8sagent/unit/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package unit_test
+
+import (
+	"testing"
+
+	gc "gopkg.in/check.v1"
+)
+
+func TestPackage(t *testing.T) {
+	gc.TestingT(t)
+}
```

Comment:
I wonder if it is worthwhile adding a test like we have in the core packages to list the juju imports.

This could help us identify if we accidentally bring in any provider  or database specific code.